object_store_factory/src/utils.rs

```diff
@@ -0,0 +1,45 @@
+use iceberg::io::{
+    S3_ACCESS_KEY_ID, S3_ALLOW_ANONYMOUS, S3_DISABLE_EC2_METADATA, S3_ENDPOINT,
+    S3_REGION, S3_SECRET_ACCESS_KEY,
+};
+use object_store::aws::AmazonS3ConfigKey;
+use std::collections::HashMap;
+use std::str::FromStr;
+
+// Go through all known keys for object store and convert them to corresponding file_io ones.
+//
+// For now only converts S3 keys.
+// TODO: At some point this should be redundant, since there is an OpenDAL adapter for object_store,
+// https://github.com/apache/iceberg-rust/issues/172
+pub fn object_store_opts_to_file_io_props(
```

Comment:
@gruuya sorry for the after merge review, you guys were faster than me.... but ..

This function should be implemented at `object_store_factory/src/aws.rs`, together with every other that deals with `aws`. 

And following the pattern and thinking about extensibility,  we need to add one at `object_store_factory/src/lib.rs` that calls it if it is for `aws`. Just like the others.

Reply:
You can put it then on lib.rs